memcache.py

```diff
@@ -500,10 +497,7 @@ def _deletetouch(self, expected, cmd, key, time=0, noreply=False):
             return 0
         self._statlog(cmd)
         extra = ' noreply' if noreply else ''
-        if time is not None and time != 0:
-            cmd = "%s %s %d%s" % (cmd, key, time, extra)
-        else:
-            cmd = "%s %s%s" % (cmd, key, extra)
+        cmd = "%s %s%s" % (cmd, key, extra)
```

Comment:
this is also used for touch, which is valid with a time value. delete could maybe just pass zero to this method instead?